qiskit/opflow/gradients/circuit_qfis/lin_comb_full.py

```diff
@@ -179,15 +190,39 @@ def convert(
 
                 # Compute −4 * Re(〈∂kψ|ψ〉〈ψ|∂lψ〉)
                 def phase_fix_combo_fn(x):
-                    return 4 * (-0.5) * (x[0] * np.conjugate(x[1]) + x[1] * np.conjugate(x[0]))
+                    return -4 * np.real(x[0] * np.conjugate(x[1]))
+
+                if phase_fix:
+                    phase_fix_op = ListOp(
+                        [phase_fix_states[i], phase_fix_states[j]], combo_fn=phase_fix_combo_fn
+                    )
+                    # Add the phase fix quantities to the entries of the QFI
+                    # Get 4 * Re[〈∂kψ|∂lψ〉−〈∂kψ|ψ〉〈ψ|∂lψ〉]
+                    qfi_ops += [SummedOp(qfi_op) + phase_fix_op]
+                else:
+                    qfi_ops += [SummedOp(qfi_op)]
 
-                phase_fix = ListOp(
-                    [phase_fix_states[i], phase_fix_states[j]], combo_fn=phase_fix_combo_fn
+            qfi_operators.append(ListOp(qfi_ops))
+
+        def check_and_realpart(qfi):
```

Comment:
Could you move this outside of the class? 🙂

Reply:
Sure. Done.